docker/dev/Dockerfile

```diff
@@ -19,6 +19,7 @@ RUN set -x && apt-get update -qq \
   libgit2-dev \
   libgraphviz-dev \
   libgtest-dev \
+  libldap2-dev \
```

Comment:
You add the LDAP dependency, but then do not configure CodeCompass with the ldap auth plugin.

Reply:
Although maybe we don't need that in the dev image.